storage/src/adb/current/mod.rs

```diff
@@ -140,3 +144,68 @@ fn verify_key_value_proof<H: CHasher, const N: usize>(
 
     reconstructed_root == *root
 }
+
+/// Return true if the given sequence of `ops` were applied starting at location `start_loc` in
+/// the log with the provided root.
+pub fn verify_range_proof<H: CHasher, O: OperationTrait, const N: usize>(
```

Comment:
We only use this in ordere.rs. I think we can use this in unordered.rs too?

Reply:
yes this was the intent all along, I think it got lost in one of my rebases. fixed.